radix-engine-interface/src/api/types/actor.rs

```diff
@@ -412,7 +413,18 @@ impl EpochManagerPackage {
                         NativeInvocation::Validator(ValidatorInvocation::Unstake(
                             ValidatorUnstakeInvocation {
                                 receiver,
-                                amount: args.amount,
+                                lp_tokens: args.lp_tokens,
+                            },
+                        ))
+                    }
+
+                    ValidatorFn::ClaimXrd => {
+                        let args: ValidatorClaimXrdMethodArgs =
+                            scrypto_decode(args).map_err(ResolveError::DecodeError)?;
+                        NativeInvocation::Validator(ValidatorInvocation::ClaimXrd(
+                            ValidatorClaimXrdInvocation {
+                                receiver,
+                                bucket: args.bucket,
```

Comment:
Can we rename it to be more specific, like `unstake_nft` ?

Reply:
Yup